packages/web/src/components/Home/SearchBar/SaveSearchButton.jsx

```diff
@@ -30,20 +30,27 @@ export default function SaveSearchButton({keyword}) {
     }
     const response = await addKeyword(keyword);
     const { success, description } = response;
+
     toast({
-      title: success ? (description === "added" ? "Subscribed!" : "Already subscribed!") : "Error!",
-      description:
-        success ? 
-        (description === "added" ?
-          "Your email was subscribed to notifications for this search." : 
-          "You are already subscribed to notifications for this search."
-        ) :
-        "Sorry, an error occurred while saving your search term. Please try again.",
-      status: success ? (description === "added" ? "success" : "info") : "error",
+      title: success
```

Comment:
might be better to unnest the conditions from the toast objects, just so it is a little cleaner. Currently, it is toast -> API check -> description "added" check (where -> means nested conditional). 

Maybe do API check -> toast -> description added? 

if you do it this way, the fail api check should be the same i believe. and then the success api check will have two different toast jsons.

its kind of like making a more efficient tree. the new way is shallower (2 levels) and the current way is deeper (like 3 levels)